src/GovUk.Education.ExploreEducationStatistics.Data.Model/LocationAttribute.cs

```diff
@@ -1,11 +1,11 @@
 #nullable enable
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
 using System;
 using System.Collections.Generic;
-using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
 
 namespace GovUk.Education.ExploreEducationStatistics.Data.Model;
 
-public abstract record LocationAttribute(string? Code, string? Name)
+public abstract record LocationAttribute(string? Code, string? Name, string? LaEstab = null)
```

Comment:
Does this need to be pushed down into the generic `LocationAttribute` superclass?  I think that it's only applicable to Schools, and I see that you have it in the `School` record type, so I would've assumed that it could just stay in that subclass.